Dockerfile

```diff
@@ -37,9 +37,11 @@ RUN	echo deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty main > /etc/apt/s
 # Packaged dependencies
 RUN apt-get update && apt-get install -y \
 	apparmor \
+	asciidoc \
 	aufs-tools \
 	automake \
 	bash-completion \
+	bsdmainutils \
```

Comment:
it looks like the only thing in the docs criu says it requires is the protobufs so what are these and `asciidoc` etc doing?